awx_collection/tests/integration/targets/host/tasks/main.yml

```diff
@@ -68,6 +78,11 @@
     that:
       - "result is changed"
 
+- name: Newly created host should have API default value for enabled
+  assert:
+    that:
+      - "lookup('awx.awx.controller_api', 'hosts/{{result.id}}/').enabled"
```

Comment:
Failure is here

Reply:
yeah you're right, I'll use the test from your patch